test/utils/initMatchers.js

```diff
@@ -5,8 +5,12 @@ import { prettyDOM } from '@testing-library/react/pure';
 import { computeAccessibleName } from 'dom-accessibility-api';
 import formatUtil from 'format-util';
 
+function isInJSDOM() {
+  return /jsdom/.test(window.navigator.userAgent);
+}
+
 function isInKarma() {
```

Comment:
The usage is specific to karma. Having a `isInBrowser = !isInJSDOM` would not be accurate.